<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistik</class>
 <widget class="QWidget" name="Statistik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RashodStatistic</string>
  </property>
  <widget class="QDateEdit" name="dateSEnd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateSStart">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetStat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>321</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Категория</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Стоимость</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Процент</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Категория</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Стоимость</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Процент</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>С</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>До</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStatAll">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Показать за все время</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Всего:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sum">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sum1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sum2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
